static/js/jira-server-url.js

```diff
@@ -55,14 +56,61 @@ const setErrorMessage = error => {
 	$("#gheServerURL").addClass("has-error");
 };
 
-/**
- * Hides the error messages
- */
 const hideErrorMessage = () => {
 	$("#gheServerURLError").hide();
 	$("#gheServerURL").removeClass("has-error");
 };
 
+const activeRequest = () => {
+	$("#gheServerBtnText").hide();
+	$("#gheServerBtnSpinner").show();
+};
+
+const requestFailed = () => {
+	$("#gheServerBtnText").show();
+	$("#gheServerBtnSpinner").hide();
+};
+
+const checkForOrCreateGitHubApp = (gheServerURL) => {
+	$.ajax({
+		type: "POST",
+		url: "/jira/app-creation",
+		data: {
+			gheServerURL
```

Comment:
Is it missing the `installationId`?

Reply:
I've created a separate issue [ARC 1465](https://softwareteams.atlassian.net/jira/software/projects/ARC/boards/385?assignee=604894ffe19f910069497c6a&selectedIssue=ARC-1465) to address this. I hadn't accounted for the need of the installationId so we're currently not passing it through from the config. Going to update that as part of that issue.